functions/index.js

```diff
@@ -1,8 +1,209 @@
+const firebase = require("firebase");
 const functions = require("firebase-functions");
+const auth = require("firebase/auth");
+const axios = require("axios");
+const { firebaseConfig } = require("firebase-functions");
+const csv = require("csvtojson");
 
 // // Create and Deploy Your First Cloud Functions
 // // https://firebase.google.com/docs/functions/write-firebase-functions
 //
-// exports.helloWorld = functions.https.onRequest((request, response) => {
-//  response.send("Hello from Firebase!");
-// });
+exports.helloWorld = functions.https.onRequest((request, response) => {
+  response.send("Hello from Firebase!");
+});
+
+const mockData = `STATE,STATE NAME,STATE OF REOPENING,NEW GATING SCORE,GATING SCORE COLOR,GATING SCORE,14-DAY TREND FOR COVID+,14-DAY TREND OF COVID+,14-DAY CHANGE IN COVID+,Rt**,SCORE,NORMALIZED,PROGRESS ON INTERVENTIONS,GRADE,CURVE,CURVED GRADE,TOTAL TESTS,% OF TESTING TARGET*,TOTAL COVID+,CASES-T-29,CASES-T-28,CASES-T-27,CASES-T-26,CASES-T-25,CASES-T-24,CASES-T-23,CASES-T-22,CASES-T-21,CASES-T-20,CASES-T-19,CASES-T-18,CASES-T-17,CASES-T-16,CASES-T-15,CASES-T-14,CASES-T-13,CASES-T-12,CASES-T-11,CASES-T-10,CASES-T-9,CASES-T-8,CASES-T-7,CASES-T-6,CASES-T-5,CASES-T-4,CASES-T-3,CASES-T-2,CASES-T-1,CASES-T-0,TOTAL DEATHS,MISSING ACTIONS,INFLUENZA-LIKE ILLNESS,COVID-LIKE SYMPTOMS (95% CI),% WHO KNOW SOMEONE WHO IS SICK,% WEARING MASKS,GATING-CRITERIA-ILI,TESTS CONDUCTED (7-DAY AVG),TESTS-T-13,TESTS-T-12,TESTS-T-11,TESTS-T-10,TESTS-T-09,TESTS-T-08,TESTS-T-07,TESTS-T-06,TESTS-T-05,TESTS-T-04,TESTS-T-03,TESTS-T-02,TESTS-T-01,TESTS-T-0,% OF TEST TARGET (US: 500K/DAY),TEST TARGET - 500K - COLOR,% OF TARGET (US: 4M/DAY),% OF TEST TARGET (INCIDENCE ADJUSTED),TEST TARGET (INCIDENT ADJUSTED),GATING CRITERIA TESTS,% POSITIVE (7-DAY AVG),POSITIVITY IS,CASE-DOUBLING-DAYS,POSITIVITY-13,POSITIVITY-12,POSITIVITY-11,POSITIVITY-10,POSITIVITY-09,POSITIVITY-08,POSITIVITY-07,POSITIVITY-06,POSITIVITY-05,POSITIVITY-04,POSITIVITY-03,POSITIVITY-02,POSITIVITY-01,POSITIVITY-00,POSITIVITY,POSITIVITY-GRADE,% OF ICU,ICU OCCUPIED,ICU COLOR,BEDS OCCUPIED,BED COLOR,RT RANGE (90% CI),DOUBLING TIME (DAYS),DEATHS-13,DEATHS-12,DEATHS-11,DEATHS-10,DEATHS-09,DEATHS-08,DEATHS-07,DEATHS-06,DEATHS-05,DEATHS-04,DEATHS-03,DEATHS-02,DEATHS-01,DEATHS-00,NEW CASES PER MILLION PER DAY,DAILY INCIDENTS PER MILLION COLOR,CURRENTLY HOSPITALIZED PER MILLION,HOSPITALIZED-13,HOSPITALIZED-12,HOSPITALIZED-11,HOSPITALIZED-10,HOSPITALIZED-09,HOSPITALIZED-08,HOSPITALIZED-07,HOSPITALIZED-06,HOSPITALIZED-05,HOSPITALIZED-04,HOSPITALIZED-03,HOSPITALIZED-02,HOSPITALIZED-01,HOSPITALIZED-00,CURRENTLY HOSPITALIZED TREND,HOSPITALIZED TREND COLOR,CASE FATALITY RATE,CASE FATALITY RATE COLOR,CONTACT TRACING POSSIBLE?,CONTACT TRACING COLOR,COVID+ RATE IS
```

Comment:
what is the mockData used for? if it's just for an example/testing should it maybe live in a different file?

Reply:
Oh, it's just the copied output from covidexitstrategy. I was only using it to prevent extra API calls when first testing (since Google charges for it). We can work on creating unit tests for it in the future. Removed.